browser/brave_rewards/rewards_tab_helper.cc

```diff
@@ -115,23 +121,27 @@ void RewardsTabHelper::WebContentsDestroyed() {
 }
 
 void RewardsTabHelper::OnBrowserSetLastActive(Browser* browser) {
+#if !defined(OS_ANDROID)
```

Comment:
you have excluded the headers for Browser here https://github.com/brave/brave-core/pull/3731/files#diff-7facd19a12f2073f3d15f0fa3181c0ffR13 but you only removed the method definition. You should either remove the entire method for android or leave the headers it needs in place.

Reply:
ok, corrected.